### ISSUE TITLE
"new" label spells out word with screenreader

### ISSUE THREAD

**Issue by laurenernest** [department-of-veterans-affairs/va.gov-team#26494]
## Summary
Discovered in #25383  - "I just tested this with VoiceOver and it reads COVID-19 vaccine just fine, but then spells out the letters "N-E-W" without a pause, where it should pause then speak the word "new"."

## Steps to Reproduce
**Test User:**  Any

1. Go to: VAOS homepage
2. Click Start scheduling button
3. On type of care page, use VoiceOver/screenreader to navigate to the COVID-19 radio option 

### Actual Result
Screenreader spells out NEW without a pause after COVID-19 vaccine

### Desired Result
Screenreader says "New type of care" then pause then "COVID-19 vaccine"
- If above isn't possible, then screenreader says "COVID-19 vaccine" then pause then "new"

## Definition of Done
- [ ] Desired result is achieved and/or document decisions made

**Comment by vbahinwillit** [comment]
Unable to reproduce. VoiceOver reads the word 'New' on my machine so the difference in behavior might be due to VoiceOver settings.

**Comment by vbahinwillit** [comment]
@jbalboni - I was using Chrome. I will try Safari as well but the big difference might be that I'm on mac  Big Sur with Peter pointed out.

**Comment by vbahinwillit** [comment]
@1Copenut - Hi Trevor. I was hoping you might be able to shed a little light on this issue. I'm unable to reproduce this on my machine which is running on mac Big Sur. Are there settings that might cause VoiceOver to read a word letter by letter as oppose to just saying the word?

Thanks

**Comment by jbalboni** [comment]
@vbahinwillit Oh, interesting. Well, we can probably fix this either way. We can mark the "new" label as aria-hidden, and then add some screen reader only text like Peter suggested in the other ticket.

I would suspect that the `text-transform: uppercase` is what is causing VO to read it as letters (or maybe that in combination with COVID in the same text element), maybe a newer version of VO on Big Sur changed that?

**Comment by laurenernest** [comment]
@vbahinwillit can you confirm if this is in Staging?

**Comment by outerpress** [comment]
Confirmed this is working on my end, thanks!
